brion/plugin/spikeReportBinary.cpp

```diff
@@ -23,116 +23,202 @@
 #include <lunchbox/pluginRegisterer.h>
 
 #include <boost/filesystem/path.hpp>
+#include <boost/filesystem.hpp>
 
-#define BINARY_REPORT_FILE_EXT ".spikes"
+#include "spikeReportTypes.h"
```

Comment:
More specific includes should come first, i.e., the includes are ordered from local...system projects.